R/settings_utils.R

```diff
@@ -112,3 +112,20 @@ pkg_name <- function(path = find_package()) {
   nm <- read.dcf(file.path(path, "DESCRIPTION"), fields = "Package")[1L]
   if (!is.na(nm)) nm
 }
+
+min_r_version <- function(path = find_package(".")) {
+  if (is.null(path)) {
+    return(getRversion())
```

Comment:
This is basically disabling the linter if no DESCRIPTION is found. I would expect a warning message detailing how to properly configure backports_linter.